libs/beacon/src/beacon_service.cpp

```diff
@@ -47,15 +48,44 @@ void TrimToSize(T &container, std::size_t max_size)
   }
 }
 
+template <typename T>
+T ChooseRandomlyFrom(T &container, std::size_t items)
```

Comment:
Worth making a common header? I am sure that I have written almost exactly the same function at some point

Reply:
Makes sense